### ISSUE TITLE
No read groups in bam

### ISSUE THREAD

**Issue by zoeward-nz** [lindenb/jvarkit#234]
I'm trying to use bamstats04 or bamstats05.
I downloaded the jar file from [https://uncloud.univ-nantes.fr/index.php/s/4sL77oWR2BFzSBH](url)

I tried running bamstats04 but it couldn't find it `[SEVERE][JvarkitCentral]jvarkit command "bamstats04" not found`
so I ran bamstats05 instead:
java -jar jvarkit.jar bamstats05 -B my.bed my.bam

I get:
[SEVERE][BamStats05]No read groups in my.bam

I was given a paired fastq to analyse which I suspect has been run over separate lanes and concatenated before being given to me.

**Comment by lindenb** [comment]
> I tried running bamstats04 but it couldn't find it

yes it's still a standalone program. I'll add to the main jvarkit in the next release.

>No read groups in my.bam

your bam is missing read groups. https://gatk.broadinstitute.org/hc/en-us/articles/360035890671-Read-groups
